package.json

```diff
@@ -113,7 +114,7 @@
     "node-polyfill-webpack-plugin": "^1.1.4",
     "postcss": "^8.4.31",
     "postcss-flexbugs-fixes": "^5.0.2",
-    "prettier": "^3.2.5",
+    "prettier": "2.2.1",
```

Comment:
So you didn't upgrade prettier in the end?

Reply:
no 😭